saos-restful-web-api/src/main/java/pl/edu/icm/saos/api/services/exceptions/ControllersEntityExceptionHandler.java

```diff
@@ -87,7 +87,7 @@
         ErrorRepresentation.Builder builder = new ErrorRepresentation.Builder();
         builder.httpStatus(errorStatus.httpStatusValue())
                 .message(ex.getMessage())
-                .name(errorStatus.errorName())
+                .httpStatusName(errorStatus.httpStatus().getReasonPhrase())
```

Comment:
Krzysiek, we shouldn't change error name to http status code name, because saos error name gives more info, for example WRONG_REQUEST_PARAMETER_ERROR tells more than BAD_REQUEST. Except that we already have httpStatus code.

What I thought of was changing the property name 'name' into something more descriptive for example 'reason':

We have now:
` {
    "error": 
    {
        "httpStatus": "400",
        "name": "WRONG REQUEST PARAMETER",
        "propertyName": "ssss",
        "message": "parameter 'ssss' : name is incorrect",
        "moreInfo": "${restful.api.error.documentation.site}WRONG_REQUEST_PARAMETER_ERROR"
    }
}
`
And it's a little confusing because we have 'name' and 'propertyName'. If we changed 'name' into e.g. 'reason' then it would be clearer I think.

` {
    "error": 
    {
        "httpStatus": "400",
        "reason": "WRONG REQUEST PARAMETER",
        "propertyName": "ssss",
        "message": "parameter 'ssss' : name is incorrect",
        "moreInfo": "${restful.api.error.documentation.site}WRONG_REQUEST_PARAMETER_ERROR"
    }
}
`